packages/web-components/src/components/va-accordion/va-accordion.tsx

```diff
@@ -58,8 +58,8 @@ export class VaAccordion {
     let headerLevel
     getSlottedNodes(clickedItem, null).map(
       (node: HTMLSlotElement) => {
-        headerText = node.innerHTML
-        headerLevel = parseInt(node.tagName.toLowerCase().split('')[1])
+        headerText = node.innerHTML !== undefined ? node.innerHTML : clickedItem.header
+        headerLevel = node.tagName !== undefined ? parseInt(node.tagName.toLowerCase().split('')[1]) : clickedItem.level
```

Comment:
If we had something like this in IE, what would happen? There isn't a `header` or `level` prop, so how would things work?

```html
<va-accordion>
  <va-accordion-item>
    <h3 slot="headline">How do I X?</h3>
    <div>Do Y then Z</div>
  </va-accordion-item>
</va-accordion>
```

Reply:
@bkjohnson so that issue you can see in the "Initial Bug" Screenshot I posted above. The first accordion item has an empty header. Which is why I had to make adjustments to the `va-accordion-item.tsx` in order to make sure if you are on IE11 the accordion item header gets included. So the updates in that file are to solve this issue.

Reply:
We're just setting these values for the analytics event, but we already have a fallback for getting the values from `clickedItem`:

https://github.com/department-of-veterans-affairs/component-library/blob/69655465824e556cc6ea5291be94557e61448986/packages/web-components/src/components/va-accordion/va-accordion.tsx#L84-L87

Could we just leave `headerText` alone here and let that `undefined || clickedItem.header` expression do the work for us?

We might get an error on `.toLowerCase()` if `node.tagName` is undefined, so if our build can compile optional chaining for `es5` could we try that here as well?